tests/console/zypper_extend.pm

```diff
@@ -65,6 +79,42 @@ sub run {
     #Remove star, don't ask for permission:
     zypper_call 'rm star';
 
+    #Install specific versioni of a package
+    my $version = script_output q[zypper se -s star |grep " star " |grep x86_64 | awk 'END {print $6}'];
```

Comment:
This can be done with one awk and `x86_64` is not needed, it should work also on another architectures or ?

Reply:
Normally, yes, I am trying it now without the grep of the archi and I will proceed with the modification ^^